config/install/views.view.tide_core_moderated_content_filters.yml

```diff
@@ -192,6 +196,132 @@ display:
           entity_type: node
           entity_field: title
           plugin_id: field
+        field_node_primary_site:
```

Comment:
hey @asarfara ,I think we might want to delete `moderation_state` filter only from the yaml file.

Reply:
Hi @vincent-gao this is moderation content view screen. This screen is different to overview content view screen.

Primary site and site filters needs to be added to Content moderation view.
Moderation state needs to be removed from Overview content view.